### PR TITLE
[stable10] reset responseXml after receiving a new response

### PR DESCRIPTION
**Pull request by individual-it**
backport #33326

### CODE REVIEW

File: tests/acceptance/features/bootstrap/BasicStructure.php [hunk 1]
```diff
@@ -626,6 +626,8 @@ public function getResponse() {
 	 */
 	public function setResponse($response) {
 		$this->response = $response;
+		//after a new response rest the response xml
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/BasicStructure.php]: s/rest/reset

**Comment by individual-it** [comment]
@phil-davis addressed the typo

**Review by phil-davis — APPROVED** [review]

**Comment by individual-it** [comment]
@ownclouders rebase

**Comment by ownclouders** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by ownclouders** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**